src/Data/CAS/RocksDB.hs

```diff
@@ -301,6 +305,44 @@ tableDelete db k = R.delete
     (encKey db k)
 {-# INLINE tableDelete #-}
 
+-- -------------------------------------------------------------------------- --
+-- Batches
+
+-- | A single update operation for a 'RocksDbTable'
+--
+data RocksDbUpdate
+    = forall k v . RocksDbDelete
```

Comment:
Duplicated foralls are noise, this can just be

```
data RocksDbUpdate k v = ...
```

Reply:
This api allows atomic updates across tables. 

Exposing the type parameters would require the use of a polymorphic list in `updateBatch`. Or it would limit all operations in a batch to operate on the same table type `[RocksDbUpdate k v]`.

I found it more convenient to just hide type parameters in the constructors for the operations.